packages/hooks/src/useIsMounted/useIsMounted.mdx

```diff
@@ -0,0 +1,68 @@
+---
+name: useIsMounted
+menu: Hooks
+route: /hooks/use-is-mounted
+---
+
+import { Playground } from "docz";
+import { useEffect, useState } from "react";
+import { Button } from "@jobber/components/Button";
+import { useIsMounted } from "@jobber/hooks";
+
+# UseIsMounted
+
+`useIsMounted` is a generalized hook that upon initial loading triggers a
+`useLayoutEffect` that sets the `isMounted.current` to true when the component
+mounts. When the component unmounts, `useLayoutEffect` calls a cleanup function
+that sets `isMounted.current` to false.
```

Comment:
I don't know that we need to dive into the implementation here so much as what is the effect of this hook and how it would be used.